lib/ast/class_def.ts

```diff
@@ -429,7 +429,7 @@ export class EntityDef extends Node {
      */
     constructor(location : SourceRange|null,
                 name : string,
-                _extends : string|null,
+                _extends : string[],
```

Comment:
I wonder if we can keep the existing constructor signature here, and automatically adjust the type when we set the property.

Reply:
fixed in c7c5506